terraform/digitalocean/addons/templates/argocd-values.yaml

```diff
@@ -0,0 +1,26 @@
+installCRDs: false
+
+server:
+  certificate:
+    enabled: true
+    domain: "domain.cluster.dev" # passed with terraform
+    issuer:
+      name: letsencrypt-prod
+      kind: ClusterIssuer
+
+  ingress:
+    enabled: true
+    annotations:
+      nginx.ingress.kubernetes.io/backend-protocol: HTTPS
+      nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+    labels: {}
+    hosts:
+      - domain.cluster.dev # passed with terraform
+    paths:
+      - /
```

Comment:
[p3] `labels` and `paths` are default and can't be changed by external configs. Why they exist in code?

Reply:
Removed from code